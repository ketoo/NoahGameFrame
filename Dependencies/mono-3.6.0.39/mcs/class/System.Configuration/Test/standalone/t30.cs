using System;
using System.Collections.Generic;
using System.Collections.Specialized;
using System.Text;
using System.Configuration;
using System.Web;
using System.Web.Configuration;

class T30
{
	static void Main(string[] args)
	{
		try
		{
			CompilationSection section = (CompilationSection)ConfigurationManager.GetSection ("system.web/compilation");

			section = (CompilationSection)ConfigurationManager.GetSection ("system.web/compilation");

			Console.WriteLine ("there are {0} assemblies listed in the section", section.Assemblies.Count);
		}
		catch (Exception e)
		{
			// Error.
			Console.WriteLine(e.ToString());
		}
	}
}
