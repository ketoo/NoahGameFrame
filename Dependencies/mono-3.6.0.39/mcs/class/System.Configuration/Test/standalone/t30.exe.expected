there are 2 assemblies listed in the section
